common/static/coffee/src/discussion/views/discussion_topic_menu_view.js

```diff
@@ -187,6 +189,38 @@
                     }
                 }
                 return name;
+            },
+
+            // TODO: this helper class duplicates functionality in DiscussionThreadListView.filterTopics
+            // for use with a very similar category dropdown in the New Post form.  The two menus' implementations
+            // should be merged into a single reusable view.
+            filterDrop: function (e) {
+                var $drop, $items, query;
```

Comment:
Note to reviewers: this method does not need a bound "this". The "this" referred to below in $items.each will be the element being iterated.